src/components/AlertRuleForm.tsx

```diff
@@ -320,3 +283,168 @@ export const AlertRuleForm = ({ rule, onSubmit }: Props) => {
     </>
   );
 };
+
+export const ProbeDurationAlertForm = ({ rule, onSubmit }: Props) => {
```

Comment:
I think all three alert forms could move into their own file. They could be grouped in a directory like we do with `CheckListItem` where we only expose what needs to be consumed externally from an `index.ts`

Reply:
they are now gone